plugins/newsdownloader.koplugin/main.lua

```diff
@@ -47,13 +47,7 @@ function NewsDownloader:addToMainMenu(menu_items)
         if not lfs.attributes(NEWS_DL_DIR, "mode") then
             lfs.mkdir(NEWS_DL_DIR)
         end
-
-        FEED_CONFIG_PATH = NEWS_DL_DIR .. FEED_CONFIG_FILE
-        if not lfs.attributes(FEED_CONFIG_PATH, "mode") then
-            logger.dbg("NewsDownloader: Creating init configuration")
-            FFIUtil.copyFile(FFIUtil.joinPath(self.path, FEED_CONFIG_FILE),
-                             FEED_CONFIG_PATH)
-        end
+	FEED_CONFIG_PATH = NEWS_DL_DIR .. FEED_CONFIG_FILE
```

Comment:
> Do you mean I should replace that with (doubled) 4 spaces?

Yes, the style guide calls for four spaces. I happen to be in the tab camp myself, but keeping things consistent really helps maintainability largely no matter what the standard is.

I apologize about the all caps. My remark was slightly contradictory to the style guide https://github.com/koreader/koreader-base/wiki/Coding-style#naming The distinction between constants and variables isn't quite the same as between globals and locals, although it probably holds up here?

> It would make more sense to use a tool than manual removal tailing whitespaces, changing tabs to 4 spaces etc? (But I guess it's a good thema for new topic)

Possibly, but you have to be very careful not to sweep actual code logic errors under the rug. It's better to warn about it and have a person decide what to do with it imo. For personal use it should be simple enough to find a script such as this one: http://lua-users.org/wiki/SourceCodeFormatter (and of course you can also run luacheck locally)